gems/pending/appliance_console.rb

```diff
@@ -193,11 +231,21 @@ module ApplianceConsole
         if agree("Apply static network configuration? (Y/N)")
           say("\nApplying static network configuration...")
 
-          Env['STATIC'] = new_ip, new_mask, new_gw, new_dns1, new_dns2
+          begin
+            eth0.address = new_ip
+            eth0.netmask = new_mask
+            eth0.gateway = new_gw
```

Comment:
Same here...can this just be:

```ruby
LinuxAdmin::NetworkInterface.new("eth0").apply_static(new_ip, new_mask, new_gw, [new_dns1, new_dns2], new_search_order)
```

Perhaps some of those are optional so maybe they are set with options instead, but you get the idea.